platform/web/js/libs/library_godot_os.js

```diff
@@ -346,6 +346,16 @@ const GodotOS = {
 		return concurrency < 2 ? concurrency : 2;
 	},
 
+	godot_js_os_pthread_pool_size_get__proxy: 'sync',
+	godot_js_os_pthread_pool_size_get__sig: 'i',
+	godot_js_os_pthread_pool_size_get: function () {
+		if (typeof PThread !== 'undefined' && PThread.runningWorkers && PThread.unusedWorkers) {
+			const total = PThread.runningWorkers.length + PThread.unusedWorkers.length;
+			return Math.max(1, total - 1);
```

Comment:
I'm being slightly defensive here by returning (total - 1) instead of the full thread count. While the sample in #104428 doesn't freeze with the full pool size, in my actual project it consistently hangs unless one thread is reserved.

Reply:
> in my actual project it consistently hangs unless one thread is reserved.
You mean without doing `Math.max(1, total - 1)`?

Reply:
Correct, didn't investigate any further being honest.